src/server/auth/cmds/cmds_test.go

```diff
@@ -351,23 +315,6 @@ func TestGetOneTimePasswordTTL(t *testing.T) {
 	require.Matches(t, "otp is invalid or has expired", errMsg.String())
 }
 
-func TestYAMLConfig(t *testing.T) {
-	if testing.Short() {
-		t.Skip("Skipping integration tests in short mode")
-	}
-	activateAuth(t)
-	defer deactivateAuth(t)
-
-	require.NoError(t, tu.BashCmd(`
-		pachctl auth get-config -o yaml \
-		  | match 'live_config_version: 1' \
-		  | match 'id_providers:' \
```

Comment:
Do you think it would make sense to keep this test but just delete the github part? I suppose this isn't like our most exhaustive test of yaml rendering, but it makes me a little anxious to remove it completely because when I've changed our yaml renderer in the past (it's wierdly hard to render and parse yaml in Go, particularly to and from protobufs because they don't generate the struct tags you need), it tended to break obscure commands.

Reply:
I moved some YAML testing into `TestConfig` so we check the JSON and YAML formats there.